src/components/Autocomplete/Autocomplete.spec.jsx

```diff
@@ -10,6 +10,17 @@ import { DropMenuDumb as DropMenu } from '../DropMenu/DropMenu';
 import * as KEYCODE from '../../constants/key-code';
 
 describe('Autocomplete', () => {
+	let requestAnimationFramePreviewValue;
+
+	beforeAll(() => {
+		requestAnimationFramePreviewValue = window.requestAnimationFrame;
+		window.requestAnimationFrame = callback => setTimeout(callback, 32);
```

Comment:
for the react 16 stuff, I'm adding this to an init script that runs before all the tests, should probably just do that here